app/build.gradle

```diff
@@ -88,6 +88,9 @@ dependencies {
     implementation "com.squareup.okhttp3:logging-interceptor:4.7.2"
     implementation 'com.github.transferwise:sequence-layout:1.1.1'
     implementation "androidx.browser:browser:1.3.0"
+
+    // https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.0.1'
```

Comment:
Hi @aniket691,
I have never used this library, but without using this also we can get the required response.
Is it really needed? What are the advantages of this can you please explain.

Reply:
We can remove it.
 It just helps give different names in model class. And we annotate and map
variable names with json object variable names.
#####################
But it helps to create response class fast.
https://www.jsonschema2pojo.org/
This tool requires this library.
And it helps to create a pojo class directly with json response in very
fast.
Just with one click.

Reply:
Hi @aniket691 we are already [using Gson library](https://github.com/adeekshith/watomatic/blob/04d1250527f2adfdfb2a51c308998b7c346e0f76/app/build.gradle#L84) in this project for JSON parsing. Can the same be used here instead of Jackson? Nothing against Jackson but just want to limit external dependencies.

But this is a great contribution. Thanks for that. We may want to improve the UI a little bit but can do in further iterations 😊

Reply:
yes
I have removed the library